tox.ini

```diff
@@ -9,6 +9,8 @@ deps =
     # we can't test eventlet>0.20.1 in our py27 CI environment until the fix
     # in https://github.com/eventlet/eventlet/issues/401 is released
     py27: eventlet==0.20.1
+    kombu3: kombu>=3.0.25,<4
+    kombu4: kombu>=4.2.0,<5
```

Comment:
As mentioned above, modern Nameko is not backwards compatible with older Kombu, so this matrix isn't required.